internal/common/enums.rs

```diff
@@ -356,6 +356,8 @@ macro_rules! for_each_enums {
                 /// The role for widget with editable text such as a
                 /// [`LineEdit`](../widgets/lineedit.md) or a [`TextEdit`](../widgets/textedit.md)
                 TextInput,
+                /// The element is a [`Switch`](../widgets/switch.md) or behaves like one.
+                Switch,
```

Comment:
The variants should not be reordered, right? Or is there a logic behind their current ordering?

Reply:
Actually, i thin it was meant to be sorted alphabetically, with the exception of `none` because that's the default, but this hasn't been respected and is not so important anyway.